maestro-cli/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
     implementation 'io.ktor:ktor-client-core:2.1.2'
     implementation 'io.ktor:ktor-client-cio:2.1.2'
     implementation 'org.rauschig:jarchivelib:1.2.0'
+    implementation 'net.harawata:appdirs:1.2.1'
```

Comment:
do we need to use a 3rd party for that? the logic there seems to be quite easy to reimplement on our own, Windows can be tricky though 🙂

Reply:
I'm fine either way the logic is simple and should be *mostly* correct for a given platform.

Reply:
Do you feel strongly about this @artem888? We can at any moment choose to refactor and write it ourselves.
The Appdirs library is tiny in code size, well written and most importantly, maintained. I'd rather use this than having to deal with feature requests for other platforms because their file system changed.

Reply:
sounds fine for me , especially if it is well-maintained.